samples/system-test/spanner.test.js

```diff
@@ -825,6 +825,25 @@ describe('Autogenerated Admin Clients', () => {
     );
   });
 
+  // dml_standard_insert_transaction_using_getTransaction
+  it('should insert rows into an example table using getTransaction method', async () => {
+    const output = execSync(
+      `node insert-query-with-get-trasanction.js ${INSTANCE_ID} ${DATABASE_ID} ${PROJECT_ID}`
```

Comment:
This can fail at random some times if the transaction is aborted by Spanner. (see also my comment above) That will make this test case a (bit) flaky.

Reply:
Removed these tests since we are not adding a sample here